src/utils/mathFormulas.js

```diff
@@ -0,0 +1,9 @@
+import _ from 'lodash';
+
+const stdev = (array) => {
+  const n = array.length;
+  const mean = _.sum(array) / n;
+  return Math.sqrt(_.sum(array.map((x) => (x - mean) ** 2)) / n);
```

Comment:
sorry, I meant creating a function called mean in the same level as stdev:
`const mean = (array) => (_.sum(array)/array.length)`

and then you can use it in both lines as follows:
`const arrayMean = mean(array);`
`return Math.sqrt(mean(array.map((x) => (x - arrayMean) ** 2)));`

Reply:
After this comment is addressed feel free to consider it approved in my end, sorry for the nitpicking

Reply:
ah I see, ok ok